google/cloud/bigtable/client.py

```diff
@@ -286,6 +292,8 @@ def get_table(
         instance_id: str,
         table_id: str,
         app_profile_id: str | None = None,
+        default_operation_timeout: float = 600,
+        default_per_request_timeout: float | None = None,
```

Comment:
I think the params for scans should be distinct from simple unary timeouts. I think maybe having:
- operation_timeout/attempt_timeout control mutate_row, check_and_mutate_row, read_modify_write, read_row
- scan_operation_timeout/scan_attempt_timeout for read_rowS
- batch_operation_timeout/attempt_operation_timeout for mutate_rowS

Reply:
yep, please leave a TODO comment though